src/components/inputs/preview.vue

```diff
@@ -96,31 +97,172 @@
+                    class=&#x22;scbd-controls km-input-lstring mb-2&#x22;&#x3E;
+                    &#x3C;slot&#x3E;&#x3C;/slot&#x3E; &#x3C;div
+                    class=&#x22;input-group mb-1&#x22; v-for=&#x22;locale in
+                    userLocales&#x22; :key=&#x22;locale&#x22;
+                    :class=&#x22;&#x60;km-input-lstring-${locale}&#x60;&#x22;
+                    &#x3E; &#x3C;input type=&#x22;text&#x22;
+                    class=&#x22;form-control&#x22;
+                    aria-describedby=&#x22;basic-addon2&#x22;
+                    v-model=&#x22;binding[locale]&#x22; :dir=&#x22;locale ==
+                    &#x27;ar&#x27; ? &#x27;rtl&#x27; : &#x27;ltr&#x27;&#x22;
+                    @update:modelValue=&#x22;emitChange&#x22; /&#x3E; &#x3C;div
+                    class=&#x22;input-group-append&#x22;&#x3E; &#x3C;button
+                    class=&#x22;btn btn-outline-primary&#x22;
+                    type=&#x22;button&#x22; id=&#x22;basic-addon2&#x22;
+                    :data-bs-toggle=&#x22;&#x27;tooltip&#x27;&#x22;
+                    :data-bs-placement=&#x22;&#x27;top&#x27;&#x22;
+                    :title=&#x22;getTerm(locale).title&#x22; &#x3E;
+                    &#x3C;/button&#x3E; &#x3C;/div&#x3E; &#x3C;/div&#x3E;
+                    &#x3C;/div&#x3E; &#x3C;/template&#x3E; &#x3C;script
+                    setup&#x3E; import { ref, watch, onMounted, defineEmits,
+                    computed } from &#x22;vue&#x22;; import { without } from
+                    &#x22;lodash&#x22;; const props = defineProps({ locales: {
+                    type: Array, required: true }, modelValue: { type: Object,
+                    default: () =&#x3E; ({}), required: true, }, disabled: {
+                    type: Boolean, default: false }, }); const emit =
+                    defineEmits([&#x22;update:modelValue&#x22;]); const
+                    userLocales = computed(() =&#x3E; props.locales); const
+                    binding = ref(props.modelValue); const emitChange = (value)
+                    =&#x3E; { emit(&#x22;update:modelValue&#x22;, value ||
+                    binding.value); }; const loadLanguages = () =&#x3E; {
+                    props.locales?.forEach((e) =&#x3E; { //TODO: call thesaurus
+                    service API }); }; const getTerm = (term) =&#x3E; { //TODO:
+                    call thesaurus service API return { title: term }; }; watch(
+                    () =&#x3E; props.locales, (newVal, oldVal) =&#x3E; { const
+                    deleted = without(oldVal, ...newVal); if (deleted?.length) {
+                    deleted.forEach((e) =&#x3E; { binding.value[e] = undefined;
+                    }); emitChange(); } loadLanguages(); } ); onMounted(()
+                    =&#x3E; { binding.value = { ...props.modelValue };
+                    loadLanguages(); }); &#x3C;/script&#x3E;
+                  </code>
                 </div>
```

Comment:
why are there changes in the inputs/preview.vue because the control has been moved to controls, please undo changes on this file

Reply:
removed the two km components from input